pulseguardian/guardian.py

```diff
@@ -283,6 +283,7 @@ def warning_email(self, users, queue_data):
 
 Make sure your clients are running correctly and are cleaning up unused
 durable queues.
+Login: https://pulseguardian.mozilla.org/queues
```

Comment:
Let's put a blank line above this, and for those who are logged in (you can try it with a github login!) that page shows the user's queues, so a different label from "Login" might be better.

Reply:
Okay, I can add a line.
Instead of Login should I put "Check messages in queue"?  Or is that too verbose?

Reply:
That's perfect!  Maybe "Check messages in the queue at <link>"

Reply:
:+1: